pystarport/default.nix

```diff
@@ -1,4 +1,4 @@
-{ pkgs ? import ../../nix { }, commit ? "" }:
+{ pkgs ? import <nixpkgs> {}, commit ? "" }:
```

Comment:
This file should be removed I think

Reply:
you mean we dont need import?

Reply:
don't need this file at all

Reply:
why rm pystarport/default.nix? I find it helpful to [gen](https://github.com/crypto-org-chain/cronos/pull/625/commits/976bb7214b9af9f5640f64f690c16b152b77e5ef) proto

Reply:
I see